src/sap.m/src/sap/m/FacetFilterList.js

```diff
@@ -9,9 +9,11 @@ sap.ui.define([
 	'sap/ui/model/ChangeReason',
 	'sap/ui/model/Filter',
 	'./FacetFilterListRenderer',
+	'./FacetFilterItem',
+	'./ListBase',
 	"sap/base/Log"
 ],
-	function(List, library, ChangeReason, Filter, FacetFilterListRenderer, Log) {
+	function(List, library, ChangeReason, Filter, FacetFilterListRenderer, FacetFilterItem, ListBase, Log) {
```

Comment:
I was actually wondering why this was done and guessed that the calls to ListBase instead of List here are intentional, a kind of trade of. If this is not the case I'll happily clean this up :+1:

Reply:
I couldn't find a reason for that. My assumption is that the methods simply have called on the object that defines them.